#! /bin/sh /usr/share/dpatch/dpatch-run
## 485129_fix_underquotation_in_m4.dpatch by Daniel Leidert (dale) <daniel.leidert@wgdd.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Satisfy aclocal complaining with:
## DP:   "warning: underquoted definition of AM_WITH_EXPAT"
## DP:
## DP: <URL:http://bugs.debian.org/485129>

@DPATCH@
diff -urNad trunk~/conftools/expat.m4 trunk/conftools/expat.m4
--- trunk~/conftools/expat.m4	2001-11-09 05:55:33.000000000 +0100
+++ trunk/conftools/expat.m4	2008-06-08 20:10:28.000000000 +0200
@@ -7,7 +7,7 @@
 dnl In addition, an Automake conditional EXPAT_INSTALLED is set accordingly.
 dnl This is necessary to adapt a whole lot of packages that have expat
 dnl bundled as a static library.
-AC_DEFUN(AM_WITH_EXPAT,
+AC_DEFUN([AM_WITH_EXPAT],
 [ AC_ARG_WITH(expat,
 	      [  --with-expat=PREFIX     Use system Expat library],
 	      , with_expat=no)
